app/models.py

```diff
@@ -86,6 +87,7 @@ def serialize(self):
             'slug': self.slug,
             'framework': self.framework,
             'status': self.status,
+            'clarification_questions_open': self.clarification_questions_open,
```

Comment:
Although we do have `one_service_limit` I think the rest are camelCase. Might be nice to fix that one up to be the same too.